lib/grpc_kit/server.rb

```diff
@@ -19,6 +19,7 @@ def initialize(interceptors: [], shutdown_timeout: 30, min_pool_size: nil, max_p
       @rpc_descs = {}
       @mutex = Mutex.new
       @stopping = false
+      @settings = []
```

Comment:
`@settings = settings` ?

Reply:
🙇  fixed a3b406f

Reply:
L22 should be fixed. not L18.

Reply:
🙇  3db6541da38f6c28861c54efda7f829c755320a4